src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -314,22 +317,25 @@ class LocalBackend(task: TaskDescriptor) extends BackendActor with StrictLogging
 
     val stderrFileLength = Try(Files.size(stderr)).getOrElse(0L)
     val failOnErrFlag = failOnStderr.getOrElse("false").toBoolean
-    val rc = returnCode.contentAsString.stripLineEnd.toInt
+    val rc = Try(returnCode.contentAsString.stripLineEnd.toInt)
 
     if (processReturnCode != 0) {
       TaskFinalStatus(Status.Failed, FailureTaskResult(
         new IllegalStateException("Execution process failed."), processReturnCode, stderr.toString))
-    } else if (failOnErrFlag && stderrFileLength > 0) {
+    } else if (rc.isFailure) { // case where docker fails.
+      TaskFinalStatus(Status.Failed, FailureTaskResult(
+        new IllegalStateException("Could not get RC file."), processReturnCode, stderr.toString))
```

Comment:
There's possibly a message in the exception associated with the failure that isn't being logged or include in the generated `IllegalStateException`.